docs/en/Repositories.md

```diff
@@ -87,6 +87,88 @@ If your entity is a soft-delete entity, you can use the `HardDeleteAsync` method
 
 See the [Data Filtering](Data-Filtering.md) documentation for more about soft-delete.
 
+## Bulk Operations
+You can execute bulk operations with `InsertMany`, `UpdateMany`, `DeleteMany` methods.
+
+Both providers `MongoDb` and `Ef Core` support bulk operations.
```

Comment:
No need to write this if both supports. Think maintainability: If we add a new provider later, then this document section will become invalid.

Reply:
Can you check Commit 6be6470f